### PR TITLE
Bugfix  - Layout iPad and division by zero (crash)

### PR DESCRIPTION
**Pull request by souzainf3**
Fixes # .

Changes proposed in this pull request:
- Menu items position on iPad
- currentPagingViewPosition() crashing
-

### CODE REVIEW

File: PagingMenuController.podspec [hunk 1]
```diff
@@ -9,13 +9,13 @@
...
   s.name             = "PagingMenuController"
-  s.version          = "0.7.9"
+  s.version          = "0.7.10b"
   s.summary          = "A paging view with customizable menu"
   s.homepage         = "https://github.com/kitasuke/PagingMenuController"
   # s.screenshots     = "www.example.com/screenshots_1", "www.example.com/screenshots_2"
   s.license          = 'MIT'
   s.author           = { "kitasuke" => "yusuke2759@gmail.com" }
-  s.source           = { :git => "https://github.com/kitasuke/PagingMenuController.git", :tag => s.version.to_s }
+  s.source           = { :git => "https://github.com/HorizonteNovasMidias/PagingMenuController.git", :tag => s.version.to_s }
```
Comment (kitasuke) [inline PagingMenuController.podspec]: Why are we using YOUR repository?

**Comment by kitasuke** [comment]
Thanks for your script, but it's for library release right? I already have fastlane actions for that case. Since I'm the only person who releases this library at this moment, I don't think it's needed.

File: PagingMenuController.podspec (hunk 1 above)
Comment (souzainf3) [inline PagingMenuController.podspec]: Sorry about the lack of attention. The private podspec was due to the urgency of the fixed bugs and we now use the fork in a private repository

File: Pod/Classes/MenuView.swift [hunk 2]
```diff
@@ -268,11 +268,22 @@ public class MenuView: UIScrollView {
...
     private func contentOffsetXForCurrentPage() -> CGFloat {
-        guard menuItemViews.count > options.minumumSupportedViewCount else { return 0.0 }
-        
-        let ratio = CGFloat(currentPage) / CGFloat(menuItemViews.count - 1)
-        return (contentSize.width - frame.width) * ratio
+        var offset: CGFloat = 0
+        if self.frame.width > self.contentSize.width {
+            return offset
+        }
+        if (currentPage == 0 && currentPage != 0) || (currentPage > 0 && currentPage != 0) {
```
Comment (kitasuke) [inline Pod/Classes/MenuView.swift]: I don't understand what your want to do, especially this part: currentPage == 0 && currentPage != 0.
Could you explain about this?

File: Pod/Classes/PagingMenuController.swift [hunk 3]
```diff
@@ -469,7 +469,7 @@ public class PagingMenuController: UIViewController, UIScrollViewDelegate {
     
     private func currentPagingViewPosition() -> PagingViewPosition {
         let pageWidth = contentScrollView.frame.width
-        let order = Int(ceil((contentScrollView.contentOffset.x - pageWidth / 2) / pageWidth))
+        let order = pageWidth <= 0 ? 0 : Int(ceil((contentScrollView.contentOffset.x - pageWidth / 2) / pageWidth))
```
Comment (kitasuke) [inline Pod/Classes/PagingMenuController.swift]: if it crashes by division by zero, is this code the only part you need to fix?

Comment (souzainf3) [inline Pod/Classes/PagingMenuController.swift]: No. This part fixes a crash that occurs in the application. The other fixes a visual bug in the iPad. As I said, we are using our fork. I found it interesting to share.

File: Pod/Classes/MenuView.swift (hunk 2 above)
Comment (souzainf3) [inline Pod/Classes/MenuView.swift]: Fix a visual bug in the iPad. When you have a few items on the screen (2 for example), they do not line up the left, are moving when they click on them.

Comment (kitasuke) [inline Pod/Classes/MenuView.swift]: This is not a bug at this moment. If you have a few items, why don't you use SegmentedControl mode?
Anyway, I don't have any plan to change this behaviour for now.

File: Pod/Classes/PagingMenuController.swift (hunk 3 above)
Comment (kitasuke) [inline Pod/Classes/PagingMenuController.swift]: Can you use `guard` to check zero? It should be checked before calculating order. 
`guard` is supposed to be for early exit.

```swift
guard pageWidth > 0 else { return 0 }
let order = Int(ceil((contentScrollView.contentOffset.x - pageWidth / 2) / pageWidth))
```

**Comment by kitasuke** [comment]
Please check my comments.
If you accept my feedbacks and remove all the changes on MenuView, podspec and Rakefile, I'll merge this PR!

**Comment by souzainf3** [comment]
Thank you. I will make corrections

**Pull request closed without merging**